data-analytics/iot-nirvana/pipeline/src/main/java/com/google/cloud/demo/iot/nirvana/common/TemperatureUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.demo.iot.nirvana.common;
+
+import com.google.gson.Gson;
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.beam.sdk.values.KV;
+
+/**
+ * Helper class to generate cities test data
+ */
+public class TemperatureUtils {
+
+  /**
+   * Create a test dataset of cities and return it as a hash map whose key is the MD5 of the
+   * latitude and longitude o the city
+   */
+  public static List<KV<String, CityTemperature>> loadCities()
```

Comment:
Return a Map instead?

Reply:
Done in commit a15f0490547bdf0943474ee656b0b380d93c2ef2